replica/database.hh

```diff
@@ -1301,6 +1301,10 @@ public:
 
         future<> add_cf(table_id id, ks_cf_t kscf, lw_shared_ptr<column_family> cf);
         future<> remove_cf(table_id id, ks_cf_t kscf) noexcept;
+        column_family& get_table(table_id id) const;
+        const table_id& get_table_id(std::pair<std::string_view, std::string_view> kscf) const;
+        std::optional<lw_shared_ptr<column_family>> get_table_if_exists(table_id id) const;
+        std::optional<table_id> get_table_id_if_exists(std::pair<std::string_view, std::string_view> kscf) const;
```

Comment:
Can simply return table_id by value.
It may bu null if not found.
It is never a valid table_id, and its bool operator will return false, same as a disengaged optional.